dbms/src/Flash/Planner/Plans/PhysicalTopN.cpp

```diff
@@ -65,6 +70,19 @@ void PhysicalTopN::buildBlockInputStreamImpl(DAGPipeline & pipeline, Context & c
     orderStreams(pipeline, max_streams, order_descr, limit, false, context, log);
 }
 
+void PhysicalTopN::buildPipelineExec(PipelineExecGroupBuilder & group_builder, Context & context, size_t /*concurrency*/)
+{
+    if (before_sort_actions && !before_sort_actions->getActions().empty())
```

Comment:
```suggestion
    if (!before_sort_actions->getActions().empty())
```
It seems that `before_sort_actions` can't be nullptr because of
```
const Block & PhysicalTopN::getSampleBlock() const
{
    return before_sort_actions->getSampleBlock();
}
```